denguefever_tw/dengue_linebot/denguebot_fsm/denguebotfsm.py

```diff
@@ -213,6 +221,44 @@ def _create_hospitals_msgs(self, hospital_list):
         hospital_messages = [template_message]
         return hospital_messages
 
+    def _create_zapper_imgmap(self, event):
+        zapper_imgmap = ImagemapSendMessage(
+            base_url='https://i.imgur.com/9piGQjS.jpg',
```

Comment:
I save this kind of constants in `constant.py`.
For style consistency, I suggest you add it there.

Reply:
Fixed